zombienet_tests/functional/0003-beefy-and-mmr.feature

```diff
@@ -6,21 +6,28 @@ Creds: config
 validator-0: is up
 validator-1: is up
 validator-2: is up
+validator-3: is up
 
 # Check authority status.
 validator-0: reports node_roles is 4
 validator-1: reports node_roles is 4
 validator-2: reports node_roles is 4
+validator-3: reports node_roles is 4
 
 # BEEFY sanity checks.
 validator-0: reports substrate_beefy_validator_set_id is 0
 validator-1: reports substrate_beefy_validator_set_id is 0
 validator-2: reports substrate_beefy_validator_set_id is 0
+validator-3: reports substrate_beefy_validator_set_id is 0
 
 # Verify voting happens and 1st mandatory block is finalized within 1st session.
 validator-0: reports substrate_beefy_best_block is at least 1 within 60 seconds
 validator-1: reports substrate_beefy_best_block is at least 1 within 60 seconds
 validator-2: reports substrate_beefy_best_block is at least 1 within 60 seconds
+validator-3: reports substrate_beefy_best_block is at least 1 within 60 seconds
+
+# Pause validator-3 and test chain is making progress without it.
```

Comment:
Based on the comment, I would expect to check block height after resume. But we actually want to see if we catchup with all BEEFY justifications.